src/consts/Stages.ts

```diff
@@ -33,76 +38,157 @@ type STAGE = STARTSTAGE | MIDSTAGE | ENDSTAGE;
 export const STAGES: ReadonlyArray<STAGE> = [
   {
     key: "Draft",
-    text: "In Draft",
+    text: "RPA Request",
     next: "PackageReview",
-    nextEventTitle: "Forward Stage Change: In Draft to Package Review",
+    nextEventTitle: "Forward Stage Change: RPA Request to Package Review",
+    readyForNext: () => true,
     previous: undefined,
     previousEventTitle: undefined,
   },
   {
     key: "PackageReview",
-    text: "Package Review/Approval",
-    next: "Post",
-    nextEventTitle: "Forward Stage Change: Package Review to Posting",
+    text: "Package Review/Concurrence",
+    next: "Recruiting",
+    nextEventTitle: "Forward Stage Change: Package Review to Recruiting",
+    readyForNext: (request) => {
```

Comment:
This should be moved down into the sub stage HRLReview, since it is at this point they would hit the "Send" button to move to Recruiting